packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -710,5 +721,4 @@ class _MotionEventsDispatcher {
 
   bool isSinglePointerAction(PointerEvent event) =>
       !(event is PointerDownEvent) && !(event is PointerUpEvent);
-}
-
+}
```

Comment:
nit: please add a [trailing newline](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file).

Reply:
done

Reply:
Seems format analyzer fails if we added a trailing newline?

Reply:
@cbracken I believe this was removing an extra blank line at the end of the file, which shouldn't be there. The trailing newline is still there.